lib/structures/GuildAuditLogEntry.js

```diff
@@ -71,11 +71,11 @@ class GuildAuditLogEntry extends Base {
                 this.membersRemoved = +data.options.members_removed;
             }
             if(data.options.type) {
-                if(data.options.type === "member") {
+                if(data.options.type === "1") {
```

Comment:
jsdoc up there says number, but this is checking for a string
Seems a bit off

```suggestion
                if(data.options.type === 1) {
```